test/basic-querying.test.js

```diff
@@ -127,7 +127,23 @@ describe('basic-querying', function() {
 
   describe('find', function() {
 
-    before(seed);
+    before(function() {
+      seed(function setupDelayingLoadedHook() {
```

Comment:
If `seed` fails, the error is silently ignored.

You can ignore this and the previous comment above if you use the approach outlined in https://github.com/strongloop/loopback-datasource-juggler/pull/864/files#r62497435.